ksql-execution/src/main/java/io/confluent/ksql/execution/codegen/SearchedCaseFunction.java

```diff
@@ -13,7 +13,7 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package io.confluent.ksql.function.udf.caseexpression;
+package io.confluent.ksql.execution.codegen;
```

Comment:
Can we move the test into the same module / package please?

Reply:
Might be worth double checking if there are any other orphaned tests left behind